hypothesis-python/src/hypothesis/extra/pandas/impl.py

```diff
@@ -177,12 +175,12 @@ def range_indexes(min_size=0, max_size=None):
 @st.cacheable
 @st.defines_strategy
 def indexes(
-    elements=None,  # type: st.SearchStrategy[Ex]
-    dtype=None,  # type: Any
-    min_size=0,  # type: int
-    max_size=None,  # type: int
-    unique=True,  # type: bool
-):
+    elements: st.SearchStrategy[Ex] = None,
+    dtype: Any = None,
```

Comment:
Why `Any` is used here when a bit about the similar `dtype` argument is annotated as `np.dtype`?

Reply:
Pandas has some "extension dtypes" that are not numpy dtypes, and I wanted to avoid over-specifying.